packages/places/__tests__/dbWritter.test.js

```diff
@@ -7,10 +12,22 @@ const newPlaceInfo = {
 };
 
 describe('Given a client that wants to save a new place', () => {
+    //the successful
     describe('When the client posts the new place info', () => {
         test('Then the postPlace method must write it on the db', async () => {
             const { dataValues } = await postPlace(newPlaceInfo);
             expect(dataValues.is_active).toBe(true);
         });
     });
+
+    //the error
+    describe('When the client does not send the required data', () => {
+        test('Then the postPlace method must not write the new row', () => {
+            try {
```

Comment:
nitpick: There is a function to test exceptions with jest, you can replace this with:
```js
expect(() => postPlace({})).rejects.toThrow();
```